### PR TITLE
systemd-link: ethtool add support for more Wake up Lan setting

### PR DESCRIPTION
**Pull request by ssahani**
Add suooport for unicast, multicast, broadcast, arp and SecureOn.

### CODE REVIEW

**Comment by poettering** [comment]
CI failed, and as it looks for a good reason. Can you look into that?

**Comment by ssahani** [comment]
Test is failing. not sure why @keszybz ?
```
duplex: -1 → (null) → -1
duplex: 0 → half → 0
duplex: 1 → full → 1
duplex: 2 → (null) → -1
     ...
wol: -1 → (null) → -1
--- stderr ---
-------
Assertion '!(i >= 0 && i < size ? sparse ? rev != i && rev != -1 : val == NULL || rev != i : val != NULL || rev != -1)' failed at ../src/shared/test-tables.h:52, function _test_table(). Aborting.
~~~

**Comment by ssahani** [comment]
Build passed @poettering

File: src/udev/net/ethtool-util.c [hunk 1]
```diff
@@ -41,9 +41,14 @@ DEFINE_STRING_TABLE_LOOKUP(duplex, Duplex);
...
-        [WOL_PHY] = "phy",
-        [WOL_MAGIC] = "magic",
-        [WOL_OFF] = "off"
+        [WOL_PHY]         = "phy",
+        [WOL_UCAST]       = "unicast",
+        [WOL_MCAST]       = "multicast",
+        [WOL_BCAST]       = "broadcast",
+        [WOL_ARP]         = "arp",
+        [WOL_MAGIC]       = "magic",
+        [WOL_MAGICSECURE] = "SecureOn",
```
Comment (poettering) [inline src/udev/net/ethtool-util.c]: oh, sorry, missed that earlier: please stick to lowercase chars for all enums. Sorry for missing that! (here and in the man page... i.e.: `"SecureOn"` → `"secureon"`

**Pull request merged**